### DISCUSSION TITLE
Decoration

### DISCUSSION THREAD

**Discussion by pashamcr** [JanDeDobbeleer/oh-my-posh#1759] [Ideas]
Is it possible to add color gradient support to oh-my-posh?
Watched an example https://fullsiteediting.com/lessons/theme-json-color-options/#h-gradients

**Comment by JanDeDobbeleer** [comment]
@pashamcr the link describes hoe to create a gradient using css, and that's not how a terminal renders colors. Is it technically possible? Yes. Every character (or background) can be colorised differently achieving a gradient BUT there's serious implications on an architecture side as we would need to track of every character printed and differentiate between ANSI and text. On top of that certain glyphs and resulting print size can influence the effect (negatively) so there are A LOT of side effects to look at that make this a pretty challenging implementation without a clear guarantee for success.

Based on that I do not see a solution (at least not an easy one).
